web/css/style.css

```diff
@@ -203,86 +403,192 @@ li .tagline {
+    font-size: .5em;
+    position: relative;
+    top: -5px;
+    left: 10px;
+}
+
+.license {
+    background-color: rgba(25, 25, 25, 0.01);
+    padding: 7px 10px;
+    font-style: italic;
+    color: rgb(100, 100, 100);
+    font-size: 0.85em;
 }
 
 /* ==|== non-semantic helper classes ======================================== */
-.ir { display: block; border: 0; text-indent: -999em; overflow: hidden; background-color: transparent; background-repeat: no-repeat; text-align: left; direction: ltr; }
-.ir br { display: none; }
-.hidden { display: none !important; visibility: hidden; }
-.visuallyhidden { border: 0; clip: rect(0 0 0 0); height: 1px; margin: -1px; overflow: hidden; padding: 0; position: absolute; width: 1px; }
-.visuallyhidden.focusable:active, .visuallyhidden.focusable:focus { clip: auto; height: auto; margin: 0; overflow: visible; position: static; width: auto; }
-.invisible { visibility: hidden; }
-.clearfix:before, .clearfix:after { content: ""; display: table; }
-.clearfix:after { clear: both; }
-.clearfix { zoom: 1; }
+.ir {
+    display: block;
+    border: 0;
+    text-indent: -999em;
+    overflow: hidden;
+    background-color: transparent;
+    background-repeat: no-repeat;
+    text-align: left;
+    direction: ltr;
+}
+
+.ir br {
+    display: none;
+}
+
+.hidden {
+    display: none !important;
+    visibility: hidden;
+}
+
+.visuallyhidden {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+}
+
+.visuallyhidden.focusable:active, .visuallyhidden.focusable:focus {
+    clip: auto;
+    height: auto;
+    margin: 0;
+    overflow: visible;
+    position: static;
+    width: auto;
+}
+
+.invisible {
+    visibility: hidden;
+}
+
+.clearfix:before, .clearfix:after {
+    content: "";
+    display: table;
+}
+
+.clearfix:after {
+    clear: both;
+}
 
+.clearfix {
+    zoom: 1;
+}
 
 /* ==|== media queries ====================================================== */
 
 @media (max-width: 800px) {
-  body {
-    margin-left: 15px;
-    margin-right: 15px;
-  }
+    body {
+        margin-left: 15px;
+        margin-right: 15px;
+    }
 }
 
 /* ==|== print styles ======================================================= */
 
 @media print {
-  * { background: transparent !important; color: black !important; text-shadow: none !important; filter:none !important; -ms-filter: none !important; }
-  a, a:visited { text-decoration: underline; }
-  a[href]:after { content: " (" attr(href) ")"; }
-  abbr[title]:after { content: " (" attr(title) ")"; }
-  .ir a:after, a[href^="javascript:"]:after, a[href^="#"]:after { content: ""; }
-  pre, blockquote { border: 1px solid #999; page-break-inside: avoid; }
-  thead { display: table-header-group; }
-  tr, img { page-break-inside: avoid; }
-  img { max-width: 100% !important; }
-  @page { margin: 0.5cm; }
-  p, h2, h3 { orphans: 3; widows: 3; }
-  h2, h3 { page-break-after: avoid; }
```

Comment:
I suggest reverting these formatting changes (you should really avoid reformatting code you don't change. I have no idea what actual changes your PR is doing on the CSS)

Reply:
Hmmm. Well, I thought CSS should follow the same indent rule as Twig templates, for there's none defined in this project, except, maybe PSR-2, that is not really intended for CSS / Js files, so...

I'll revert to the previous spacing.

Reply:
Though, actually, I think it's also a good idea to get new formatting.

This file had at least three different code styles into it. We would never allow that on PHP/Twig, right? So why we would postpone indefinitely the CSS / JS to be heterogenous?

I changed the indenting back to two spaces, but I really think we should move on this code to be following one code style. That would be more suitable for the future.

Reply:
Changing code styles should be discussed with maintainers first. And it should not be done in a PR doing other stuff (and even less in the same commit), as that hides what the commit does. Here, maintainers have to guess which lines are just reformatted and which lines are actually changed.

and the different styles are because this file contains some third-party CSS (the normalize.css one). There is no build pipeline to build the final CSS at deploy time, which is why it is in the same file (and it is kept compact, as this part of the CSS will never be edited in this repo, and this makes the CSS file smaller).

Reply:
Yeah, I guessed why, but I thought that this was a good opportunity to do so. My bad.

Reply:
Update: reverted the normalize part.